mmdet/models/data_preprocessors/track_data_preprocessor.py

```diff
@@ -45,22 +48,57 @@ class TrackDataPreprocessor(DetDataPreprocessor):
             Defaults to False.
         rgb_to_bgr (bool): whether to convert image from RGB to RGB.
             Defaults to False.
+        use_det_processor: (bool): whether to use DetDataPreprocessor
+            in trainging phrase. Defaults to False.
```

Comment:
```suggestion
            in training phrase. This is mainly for some tracking models fed into one image rather than a group of image in training. Defaults to False.
```